### ISSUE TITLE
Test the new color system

### ISSUE THREAD

**Issue by anitadesigns** [hackforla/expunge-assist#1341]

**Comment by sylvia-nam** [comment]
@anitadesigns - to clarify, is this issue for when the new colors are implemented by Design and then require A/B testing (or something else) by UXR?

**Comment by anitadesigns** [comment]
@sylvia-nam that's correct! I made it pretty general so that UXR can decide the best research method to implement

**Comment by sylvia-nam** [comment]
Added #1228 as connected issue (rather than as dependency) following 5/13 discussion at PM-Leads meeting.

**Comment by sylvia-nam** [comment]
Sara will draft usability testing protocol to test new color system so that it aligns with her research findings and Content's Voice and Tone: https://github.com/hackforla/expunge-assist/issues/618#issuecomment-2000005965

> Supportive - warmth and intention (vs empowering and helpful), supportive helps them grow which is empowering, like teacher or mentor rooting you on
> Straightforward - less technical language is more approachable, good for UX writing in general, ease of use, accessibility - for reading levels of users, and ESL, telling the user what they need to know without burdening them
> Trustworthy - (vs secure, which lacked warmth), applies to both site and process, has warmth, reliable

**Comment by anitadesigns** [comment]
Added a dependency to this issue since Design team needs to adjust and standardize the color system before they can be tested

**Comment by raeraeicky** [comment]
Moved to Icebox as this is dependent on [1228](https://github.com/hackforla/expunge-assist/issues/1228) being completed
